src/Command/MakeSerializerEncoderCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Command;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Serializer\Serializer;
+
+/**
+ * @author Piotr Grabski-Gradzinski <piotr.gradzinski@gmail.com>
+ */
+final class MakeSerializerEncoderCommand extends AbstractCommand
+{
+    protected static $defaultName = 'make:serializer:encoder';
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Creates a new custom encoder class')
```

Comment:
What about:

```
Creates a new serializer encoder class
```

Reply:
Yup, fixed in command description, success message (as is :) ) and command help.